docs/Variables.adoc

```diff
@@ -30,13 +33,21 @@ Required variable, but may be set to `none` where not applicable.
 | Name
 | Description
 
+| `pass:[__meta__]`
+| Metadata used by the Babylon infrastructure when running AgnosticD.
+This variable should be ignored by workloads, configs, and roles.
```

Comment:
Good :+1: 
So one could ask, what is it used for ?

Reply:
One could ask, but the answer should be go look at the Babylon docs, it isn't used by agnosticd.